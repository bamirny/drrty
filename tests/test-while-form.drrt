def int main(){
	int i;
	bool b;
	
	b = True;
	i = 0;
	
	makeHeader("Log the names and numbers below!");

	while(i<6){
		if(b == True){
			makeText("Name");
			makeInput("");
			createElement("button","Submit","");
			i = i+1;
			b = False;
		}
		else{
			makeText("Phone Number");
			makeInput("");
			createElement("button","Submit","");
			i = i+1;
			b = True;
		}
	}
	return 0;
	
}