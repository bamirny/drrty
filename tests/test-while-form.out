<h1>Log the names and numbers below!</h1>
<p>Name</p>
<input class="">
<button class="">Submit</button>
<p>Phone Number</p>
<input class="">
<button class="">Submit</button>
<p>Name</p>
<input class="">
<button class="">Submit</button>
<p>Phone Number</p>
<input class="">
<button class="">Submit</button>
<p>Name</p>
<input class="">
<button class="">Submit</button>
<p>Phone Number</p>
<input class="">
<button class="">Submit</button>
